railties/lib/rails/tasks/yarn.rake

```diff
@@ -3,7 +3,7 @@
 namespace :yarn do
   desc "Install all JavaScript dependencies as specified via Yarn"
   task :install do
-    system("./bin/yarn install --no-progress --production")
+    system("./bin/yarn install --no-progress --freeze-lockfile --production")
```

Comment:
The option is `—frozen-lockfile`, not `—freeze-lockfile`.

Reply:
Fixed in https://github.com/rails/rails/commit/f9ec3e5cc3753b7a46cc1a34d57ca941e4cf7211